src/features/profile/DangerCard.js

```diff
@@ -61,7 +67,10 @@ const DangerCard = () => {
       if (error) {
         alert(`Oopsie (${error.message}), please try again.`)
       } else {
+        // call trackGoal here, when the customer has succeeded in deleting her/his/their entries
+        trackGoal(FATHOM_DELETE_CONFIRMED);
```

Comment:
**suggestion:** Did not see this until now, but has to be FATHOM_DELETE_ALL_ENTRIES_CONFIRMED

Reply:
I have added it to Netlify as GATSBY_FATHOM_DELETE_ALL_ENTRIES_CONFIRMED, so should show in console.log as soon as this is changed and the PR builds again.

Reply:
FATHOM_DELETE_ALL_ENTRIES_CONFIRMED #303
942cc32

Reply:
FATHOM_DELETE_ALL_ENTRIES_CONFIRMED #303
05b6b9a